README.md

```diff
@@ -29,3 +29,34 @@ Storage layer with super easy to use in-memory data views and Sherlock-based per
 
 #### [Type Evolution](https://github.com/C5T/Current/blob/master/TypeSystem/Evolution.md)
 Compact and autogenerated C++ framework to evolve objects from one type hierarchy into another type hierarchy while remaining fully within the strong typing paradigm.
+
+## Contribution
+
+Welcome, contributors! Please [start here by signing the CLA](https://github.com/C5T/Current/blob/master/contributors/README.md).
+
+## Quick Start
+
+Install the development dependencies (see below).
+
+Clean, build and run all tests, verify code:
+```
+make clean
+make test
+make check
+```
+
+Build and run tests for each module separately:
+```
+make individual_tests
+```
+
+Build and run tests for one of the modules (e.g. `Blocks/HTTP`):
+```
+( cd Blocks/HTTP && make -j test )
+```
+
+## Development Dependencies
+
+- `nasm` for `FnCAS`. **macOS**: `brew install nasm`
```

Comment:
Perhaps add some "or disable FnCAS tests" comment? Perhaps a macro and an environmental var for it, as we're on this page?